src/pages/Resources/index.tsx

```diff
@@ -53,12 +53,12 @@ const linkProps: AppLinkProps[] = [
   }
 ];
 
-const AppsPageComponent: React.SFC = () => (
+const ResourcesPageComponent: React.SFC = () => (
   <React.Fragment>
     <div className="content">
       <div className="apps">
-        {linkProps.map((props: AppLinkProps, idx: number) => (
-          <AppLink key={idx} {...props} />
+        {linkProps.map((props: ResourceLinkProps, idx: number) => (
+          <ResourceLink key={idx} {...props} />
         ))}
         <div className="app-page-mobile-padding" />
```

Comment:
Change this classname as well.
```suggestion
         <div className="resource-page-mobile-padding" />
```